### ISSUE TITLE
Ability to change Genius Controller operating mode from xlights.

### ISSUE THREAD

**Issue by MDickson00** [xLightsSequencer/xLights#4823]
This may fall under the BUG category as well as an enhancment.

Currently in xlights if running a Genius (this may apply to other controllers) controller in remote mode (Sync Packets), when you upload outputs from xlights it changes the Operating Mode back to whatever is selected in xlights (E 1.31/DDP/Artnet).

Can the option to change the Operating Mode over to SD Card (Standalone) or SD Car (Sync Packets) be added in the xlights controller settings? 

See images below.
![Screenshot 2024-09-22 10 31 13](https://github.com/user-attachments/assets/c3a6940f-71d0-4781-b282-68bbf8ec1927)
![2024-09-22 10_38_36-Window](https://github.com/user-attachments/assets/0f431a05-d9a1-41e5-b945-61c53e058b9d)

**Comment by cybercop23** [comment]
Won't this same thing apply to a Falcon too? Be able to change its play method?
IDK if it doesn already, but I'd think this must first be supported by the ctrl firmware before we can direct/force any changes.

**Comment by MDickson00** [comment]
Yes this may be useful for Falcon controllers as well, it seems reasonable that it would be possible since you can change between E1.31/Arnet/DDP. The problem is, if you manually set a controller to SD (Sync Packets) it gets changed back to DDP/Artnet/E1.31 because it is selected in the controller settings.

Maybe at the very least have a blank option in xlights so it simply ignores the Operating Mode and doesn't change it.

**Comment by computergeek1507** [comment]
I had some discussions with David on how to fix this, if he should change the firmware to allow bridging and remote mode. I don't recall if would be possible for the firmware to allow both. I know the Ethernet chip has limitations on what port configuration. If you unlink the input/output upload it shouldnt change it. I believe Falcon has the same issue.

**Comment by MDickson00** [comment]
Unlinking the input/output seems easiest, didn't consider that.

**Comment by computergeek1507** [comment]
It wouldn't be hard to add a Remote/Master input type or maybe skip that setting if set to "xLights only"

**Comment by cybercop23** [comment]
PR Ready to persist mode if Standalone or Remote when pushing from xLights. If in the other modes, it will swich it based on what xLights has configured. Again, if configured as DPP but ctrl is in Remote mode, it will update config to be DDP based and put it back in Remote moode.
